qx/handle_base58.go

```diff
@@ -63,9 +63,7 @@ func Base58CheckEncode(version []byte, mode string, hasher string, cksumSize int
 	}
 	// Show the encoded data.
 	//fmt.Printf("Encoded Data ver[%v] : %s\n",ver, encoded)
-	if err != nil {
-		fmt.Errorf("%s", err)
-	} else {
+	if err == nil {
```

Comment:
Why need to change the qx , how qx affects the p2p?

Reply:
![image](https://user-images.githubusercontent.com/35813306/99865351-edac6180-2be3-11eb-9eec-8ad58e15467a.png)

Reply:
oh, my bad at https://github.com/Qitmeer/qitmeer/commit/e88f1fc0d953417e2bbdaab3c97da5523e4a7b1b
should change to
```golang
if err != nil {
	ErrExit(err)
}
fmt.Printf("%s\n", encoded)
```

Reply:
https://github.com/Qitmeer/qitmeer/pull/400/commits/f2b2212bc0161ba6b0c0ab621e934c612c0d6308